### DISCUSSION TITLE
SOS

### DISCUSSION THREAD

**Discussion by Burning1010** [Chia-Network/chia-blockchain#17686] [Support]
Hello. Sent 2 chia coins to gate.io exchange block confirmed 5034496.
The coin has not reached the exchange. What to do? Please help ! Thank you!

**Comment by BrandtH22** [comment]
Hey @Burning1010 , I would confirm the xch address that was used to send funds to the exchange and if correct I would confirm with the exchanges customer support as to why the funds are not visible. 

Side note - exchanges have not integrated the clawback capabilities so if your funds were sent with a clawback you as the sender will need to claw them back then resend them without a clawback.

We generally can provide more timely and thorough support in our discord server (https://discord.gg/chia). 

Let us know if any questions arise 👍

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
